modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1908,6 +1908,43 @@ public void testGetObjectEntriesAggregationFacets() throws Exception {
 		Assert.assertEquals(facetValue.getNumberOfOccurrences(), (Integer)2);
 	}
 
+	@Test
+	public void testGetObjectEntriesAggregationManyToManyRelationship()
+		throws Exception {
+
+		ObjectDefinition objectDefinition = _createObjectDefinition(
+			Arrays.asList(
+				new TextObjectFieldBuilder(
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).name(
+					"textObjectFieldName"
+				).build()));
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				_adminUser.getUserId(),
+				objectDefinition.getObjectDefinitionId(),
+				objectDefinition.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"manyToManyRelationshipName",
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		_addAggregationObjectField(
+			"SUM", objectDefinition.getObjectDefinitionId(), "id",
+			objectRelationship.getName());
```

Comment:
@pedro-oliveira446 maybe we should move this test to `testAddObjectEntry` and group it by:

`// Aggregation field without filters based on Many to Many relationship`

could you please try to move it to `testAddObjectEntry`?

Thanks!